students/Jean-Baptiste/lessons/lesson03/assignment03_solution_JB/test_basic_operations.py

```diff
@@ -0,0 +1 @@
+
'''This is to test the basic operations '''
from basic_operations import add_customer
from customers_model import Customer
import pytest
ok_customer = {'customer_id': 'ABCD11100','name': 'Jack','last_name': 'Smith','home_address':
'123 U-village, Seattle, WA 98105', 'phone_number': '425-1234-4567','email_address': 'Jacksmith@hotmail.com',
'status': True,'credit_limit': '40'}
def test_add_ok_customer():
    add_customer(**ok_customer)
    test_customer = Customer.get(Customer.customer_id==ok_customer['customer_id'])
    assert test_customer.email_address == ok_customer['email_address']

```

Comment:
When git asks you if it should change line endings to unix form, say yes, then the ^M will go away, and formatting will look normal.